hibernate-core/src/main/java/org/hibernate/metamodel/mapping/MappingModelHelper.java

```diff
@@ -27,80 +16,79 @@ public class MappingModelHelper {
-			}
-			else {
-				qualifier = tableGroup.resolveTableReference( basicPart.getContainingTableExpression() ).getIdentificationVariable();
-			}
-			if ( sqlExpressionResolver == null ) {
-				return new ColumnReference(
-						qualifier,
-						basicPart
-				);
-			}
-			else {
-				return sqlExpressionResolver.resolveSqlExpression(
-						createColumnReferenceKey( qualifier, basicPart ),
-						sqlAstProcessingState -> new ColumnReference(
-								qualifier,
-								basicPart
-						)
-				);
-			}
-		}
-	}
+//
+//	public static Expression buildColumnReferenceExpression(
+//			ModelPart modelPart,
+//			SqlExpressionResolver sqlExpressionResolver) {
+//		return buildColumnReferenceExpression( null, modelPart, sqlExpressionResolver );
+//	}
+//
+//	public static Expression buildColumnReferenceExpression(
+//			TableGroup tableGroup,
+//			ModelPart modelPart,
+//			SqlExpressionResolver sqlExpressionResolver) {
+//		final int jdbcTypeCount = modelPart.getJdbcTypeCount();
+//
+//		if ( modelPart instanceof EmbeddableValuedModelPart ) {
+//			final List<ColumnReference> columnReferences = new ArrayList<>( jdbcTypeCount );
+//			modelPart.forEachSelectable(
+//					(columnIndex, selection) -> {
+//						final ColumnReference colRef;
+//						final String qualifier;
+//						if ( tableGroup == null ) {
+//							qualifier = selection.getContainingTableExpression();
+//						}
+//						else {
+//							qualifier = tableGroup.resolveTableReference( selection.getContainingTableExpression() ).getIdentificationVariable();
+//						}
+//						if ( sqlExpressionResolver == null ) {
+//							colRef = new ColumnReference(
+//									qualifier,
+//									selection
+//							);
+//						}
+//						else {
+//							colRef = (ColumnReference) sqlExpressionResolver.resolveSqlExpression(
+//									createColumnReferenceKey( qualifier, selection ),
+//									sqlAstProcessingState -> new ColumnReference(
+//											qualifier,
+//											selection
+//									)
+//							);
+//						}
+//						columnReferences.add( colRef );
+//					}
+//			);
+//			return new SqlTuple( columnReferences, modelPart );
+//		}
+//		else {
+//			assert modelPart instanceof BasicValuedModelPart;
+//			final BasicValuedModelPart basicPart = (BasicValuedModelPart) modelPart;
+//			final String qualifier;
+//			if ( tableGroup == null ) {
+//				qualifier = basicPart.getContainingTableExpression();
+//			}
+//			else {
+//				qualifier = tableGroup.resolveTableReference( basicPart.getContainingTableExpression() ).getIdentificationVariable();
+//			}
+//			if ( sqlExpressionResolver == null ) {
+//				return new ColumnReference(
+//						qualifier,
+//						basicPart
+//				);
+//			}
+//			else {
+//				return sqlExpressionResolver.resolveSqlExpression(
+//						createColumnReferenceKey( qualifier, basicPart ),
+//						sqlAstProcessingState -> new ColumnReference(
+//								qualifier,
+//								basicPart
+//						)
+//				);
+//			}
+//		}
+//	}
+//
```

Comment:
Should be deleted? There's git history if we ever want that code back.

Reply:
Dunno, I don't really like hard-deleting things I don't understand. All I know is it was unused...